### ISSUE TITLE
Upstream Headroom hooks config is rejected by Codex hooks parser

### ISSUE THREAD

**Issue by styler-ai** [styler-ai/ProjectAtlas#234]
## Summary
ProjectAtlas post-release Codex smoke emits a plugin hook parse warning for the Headroom plugin:

```text
warning: failed to parse plugin hooks config <CODEX_HOME>/plugins/cache/headroom-marketplace/headroom/0.22.3/hooks/hooks.json: unknown field `description`, expected `hooks` at line 2 column 15
```

The local hooks file includes a top-level `description` field next to `hooks`:

```json
{
  "description": "Headroom plugin hooks — ensure the local Headroom runtime is available for initialized agents.",
  "hooks": { }
}
```

## Expected behavior
The installed plugin hook config should match the Codex hooks schema, or the Codex parser should tolerate/document supported metadata fields.

## ProjectAtlas impact
The warning appears during ProjectAtlas smoke runs and can obscure ProjectAtlas plugin/runtime validation output even though it is not caused by ProjectAtlas.

## Acceptance criteria
- Confirm this is upstream/non-ProjectAtlas ownership or route it to the responsible upstream plugin.
- ProjectAtlas release-smoke notes classify this warning as external if it still appears.
- The warning no longer blocks or confuses ProjectAtlas install/update validation.

**Comment by styler-ai** [comment]
Closing as not planned for ProjectAtlas: this warning is for the upstream Headroom hooks config, not a ProjectAtlas-owned hooks file. ProjectAtlas indexed search found no Headroom or hooks.json references in this repository, so there is no local code path to fix.
